src/events/events.ts

```diff
@@ -1,258 +1,74 @@
-import type {
-  APIGatewayProxyEvent,
-  APIGatewayProxyEventV2,
-  AppSyncResolverEvent,
-  DynamoDBStreamEvent,
-  EventBridgeEvent,
-  KinesisStreamEvent,
-  S3Event,
-  SNSEvent,
-  SQSEvent,
-} from 'aws-lambda';
-
-import {
-  isStepFunction,
-  LUMIGO_EVENT_KEY,
-  md5Hash,
-  recursiveGetKey,
-  safeExecute,
-  STEP_FUNCTION_UID_KEY,
-} from '../utils';
-import type {
-  ApiGatewayV1EventData,
-  ApiGatewayV2EventData,
-  AppSyncEventData,
-  DynamoDBStreamEventData,
-  EventBridgeEventData,
-  EventData,
-  EventInfo,
-  IncomingEvent,
-  IncomingEventRecord,
-  KinesisStreamEventData,
-  S3EventData,
-  SNSEventData,
-  SQSEventData,
-  StepFunctionEventData,
-} from './event-data.types';
-import { EventTrigger } from './event-trigger.enum';
-
-export const getTriggeredBy = (event: IncomingEvent): EventTrigger => {
```

Comment:
I split the code from this file between different files - per resource. 
Moved it to `Handlers` design pattern, and separated the logic to `should` and `handle`